scala-hello-world/src/test/scala/com/labs1904/hwe/exercises/ChallengeTests.scala

```diff
@@ -0,0 +1,133 @@
+package com.labs1904.hwe.exercises
+
+import org.scalatest.FunSpec
+
+
+class ChallengeTests extends FunSpec{
```

Comment:
Could we rename this to `ChallengeProblemsTests`? Generally I try to to follow the convention where if the class is named `Blah` then the tests are in a file named `BlahTests`.